File: 1010.java
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) {
 
        Scanner scanf = new Scanner(System.in);      
        


        int cod1 = scanf.nextInt();
        int n1 = scanf.nextInt();
        float valor1 = scanf.nextFloat();
        int cod2 = scanf.nextInt();
        int n2 = scanf.nextInt();
        float valor2 = scanf.nextFloat();
        
        float total = (n1 * valor1) + (n2 * valor2);

        System.out.printf("VALOR A PAGAR: R$ %.2f\n", total);
 
    }
 
}

File: Iniciante/1010.py
cod1, n1, valor1 = input().split(" ")

cod1 = int(cod1)
n1 = int(n1)
valor1 = float(valor1)

cod2, n2, valor2 = input().split(" ")

cod2 = int(cod2)
n2 = int(n2)
valor2 = float(valor2)

total = (n1 * valor1) + (n2 * valor2)

print(f'VALOR A PAGAR: R$ {total:.2f}')